WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/ui/WatchDogView.java

```diff
@@ -113,50 +167,94 @@ private void createActiveView() {
 		container = UIUtils.createGridedComposite(oneColumn, 2);
 		container.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
 
-		createSWTChart(
-				container,
-				createBarChart(createDevelopmentBarDataset(),
-						"Your Development Activity", "", "minutes"));
-		createSWTChart(
-				container,
-				createPieChart(createDevelopmentPieDataset(),
-						"Your Development Activity"));
+		createSWTChart(container, createBarChart(createDevelopmentBarDataset(),
+				"Your Development Activity", "", "minutes"));
+		createSWTChart(container, createPieChart(createDevelopmentPieDataset(),
+				"Your Development Activity"));
 		UIUtils.createLabel("", container);
 		UIUtils.createLabel("", container);
 
-		createSWTChart(
-				container,
+		createSWTChart(container,
 				createBarChart(createProductionVSTestBarDataset(),
 						"Your Production vs. Test Activity", "", "minutes"));
-		createSWTChart(
-				container,
+		createSWTChart(container,
 				createPieChart(createProductionVSTestPieDataset(),
 						"Your Production vs. Test Activity"));
 
 		UIUtils.createLabel("", container);
 		UIUtils.createLabel("", container);
 
-		createSWTChart(
-				container,
+		createSWTChart(container,
 				createPieChart(createPerspectiveViewPieDataset(),
 						"Your Perspective Activity"));
-		createSWTChart(
-				container,
+		createSWTChart(container,
 				createStackedBarChart(createJunitExecutionBarDataset(),
 						"Your Test Run Activity", "", ""));
+		UIUtils.createLabel("", container);
+		UIUtils.createLabel("", container);
+
+		if (selectedDebugInterval != null) {
+			createDebugIntervalSelectionList();
+			debugChartContainer = UIUtils.createGridedComposite(oneColumn, 1);
+			debugChartContainer.setLayoutData(
+					new GridData(SWT.FILL, SWT.FILL, true, true));
+			createSWTChart(debugChartContainer, createDebugEventGanttChart());
+		}
 
 		createShowingStatisticsLine();
 		createTimeSpanSelectionList();
 	}
 
+	private JFreeChart createDebugEventGanttChart() {
+		eventStatistics = new EventStatistics(
+				InitializationManager.getInstance().getDebugEventManager(),
+				selectedDebugInterval);
+		final GanttCategoryDataset dataset = eventStatistics
+				.createDebugEventGanttChartDataset();
+
+		final JFreeChart chart = ChartFactory.createGanttChart(
+				"Debug Events During Selected Debug Interval", // chart title
```

Comment:
Not from its JavaDoc, as there isn't any, but the are actually clear from the parameter names. So, I'll remove the comment.